Sources/PointFree/Episode/Episode.swift

```diff
@@ -101,11 +103,16 @@ private let topLevelTagsView = View<[Tag]> { tags in
     ])
 }
 
-private let episodeTocView = View<[Episode.TranscriptBlock]> { blocks in
+private let episodeTocView = View<[TranscriptBlock]> { blocks in
   blocks
-    .filter { $0.type == .title && $0.timestamp != nil }
-    .flatMap { block in
-      tocEntryView.view((block.content, block.timestamp ?? 0))
+    .flatMap { block -> [Node] in
```

Comment:
this `-> [Node]` seems to be required, otherwise Swift tries to infer this as an `-> B?` flat map :/